### ISSUE TITLE
Implement the "table-active" class

### ISSUE THREAD

**Issue by caohanh1502** [vinorodrigues/bootstrap-dark-5#25]
Bootstrap 5 has "table-active" class to highlight a table row or cell:
https://getbootstrap.com/docs/5.0/content/tables/#active-tables

However in dark mode, the normal "table" is missing the background-color of the active row (or cell):
![image](https://user-images.githubusercontent.com/7068354/123011148-113dc180-d3ea-11eb-9d31-cc30da7e4db8.png)

(Btw, it works fine on the dark table "table table-dark").

Edited: Now I notice that [Striped rows](https://getbootstrap.com/docs/5.0/content/tables/#striped-rows) also does not work on a normal "table", but works fine on "table table-dark".

**Comment by vinorodrigues** [comment]
Yes ... my fault.  Kind of stupid of me ... code was in `_variables-alt.scss`:

```scss
$table-active-bg-alt:  rgba($black-alt, $table-active-bg-factor-alt) !default;
```

So that generated a black semi-opaque row... but on a back background... that's just back.   Duh!

Fixed this to:
```scss
$table-active-bg-alt:  rgba($white-alt, $table-active-bg-factor-alt) !default;
```

*(Look out for the fix in release: v1.0.2 to match BS v5.0.2)*

**Comment by vinorodrigues** [comment]
Fixed in release 1.0.2
